.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+### Type:
```

Comment:
The idea is to have a common PR template for the whole organization, but we can keep this one until the template is ready!